### ISSUE TITLE
How to specify the validation data during training and output test accuracy

### ISSUE THREAD

**Issue by yghlc** [TheLegendAli/DeepLab-Context#30]
Hello,
    There is no test net accuracy in the log file even I specified the test data layer in the train.prototxt file: 
```
layers {
  name: "data"
  type: IMAGE_SEG_DATA
  top: "data"
  top: "label"
  image_data_param {
    root_folder: ""
    source: "./list/test_aug.txt"
    label_type: PIXEL
    batch_size: 100
  }
  transform_param {
    mean_value: 75.209
    mean_value: 85.950
    mean_value: 95.685
    crop_size: 663
    mirror: false
  }
  include: { phase: TEST }
}
```
Only loss and train net accuracy in the log file, such as: 
```
I0424 09:32:20.303715 32660 solver.cpp:209] Iteration 50, loss = 0.270298
I0424 09:32:20.303766 32660 solver.cpp:224]     Train net output #0: accuracy = 0.883158
I0424 09:32:20.303774 32660 solver.cpp:224]     Train net output #1: accuracy = 0.780627
I0424 09:32:20.303809 32660 solver.cpp:224]     Train net output #2: accuracy = 0.70205
```
How do I get the test accuracy during training?  
` What the meaning of Train net output #0, Train net output "#1", Train net output "#2"?  "#0", "#1", "#2" means something?`

Thanks for any suggestion!

**Comment by mwangaz** [comment]
Hey. Did you figure out how to get the validation accuracy or loss? I also want to do that.
